### PR TITLE
Add Lambda Labs catalog

### PR DESCRIPTION
**Pull request by ewzeng**
Adding Lambda Labs catalog! @Michaelvll @concretevitamin

### CODE REVIEW

File: catalogs/v5/lambda/vms.csv [hunk 1]
```diff
@@ -0,0 +1,122 @@
+InstanceType,AcceleratorName,AcceleratorCount,vCPUs,MemoryGiB,Price,Region,GpuInfo,SpotPrice
+gpu_8x_v100,V100,8.0,92.0,448,4.4,australia-southeast-1,"{'Gpus': [{'Name': 'V100', 'Manufacturer': 'NVIDIA', 'Count': 8.0, 'MemoryInfo': {'SizeInMiB': 16384}}], 'TotalGpuMemoryInMiB': 16384}",
```
Comment (WoosukKwon) [inline catalogs/v5/lambda/vms.csv:2]: While it is not required, why don't we make `MemoryGiB` as float, just for safety?

**Review by WoosukKwon — APPROVED** [review]
LGTM. Please check out my minor comment.

**Comment by WoosukKwon** [comment]
Don't we want to put the catalog in v6? Otherwise, the catalog will not be automatically downloaded to current users (unless they delete the `~/.sky/catalogs/v5` directory by themselves).

File: catalogs/v5/lambda/vms.csv (hunk 1 above)
Comment (ewzeng) [reply catalogs/v5/lambda/vms.csv:2]: Sure. That sounds good!

**Comment by ewzeng** [comment]
> Don't we want to put the catalog in v6? Otherwise, the catalog will not be automatically downloaded to current users (unless they delete the `~/.sky/catalogs/v5` directory by themselves).

I did not know this. Should I move AWS/Azure/GCP to v6 as well?

**Comment by WoosukKwon** [comment]
Sorry, @ewzeng  I was wrong. It should be automatically downloaded, since it's a new catalog. Never mind.

File: catalogs/v5/lambda/vms.csv (hunk 1 above)
Comment (ewzeng) [reply catalogs/v5/lambda/vms.csv]: Fixed!

**Comment by ewzeng** [comment]
I have changed `MemoryGiB` to float. How does it look now?

(I also don't have push access. You might need to squash + merge for me).

**Comment by WoosukKwon** [comment]
LGTM. Let's merge it.

**Pull request merged**